src/bp/nlu-core/tools/math.test.ts

```diff
@@ -0,0 +1,64 @@
+import { computeNorm, computeQuantile, scalarMultiply, vectorAdd } from './math'
+
+describe('Math utils', () => {
+  const vec0 = []
+  const vec1 = [1, 1, 1]
+  const vec2 = [2, 2, 2]
+  const vec3 = [4, 5, 6]
+
+  test('VectorAdd', () => {
+    expect(() => vectorAdd(vec0, vec2)).toThrow()
```

Comment:
I know this should be done in another PR, but just so you know. We added math.js library which include those utils (but faster) after we implemented those. Can we add a TODO comment to remove implementation and tests

Reply:
yeah sure